pkgs/development/tools/language-servers/dcd/makefile.diff

```diff
@@ -0,0 +1,14 @@
+--- a/dcd/makefile
++++ b/dcd/makefile
+@@ -13,11 +13,8 @@
+ DPARSE_DIR := libdparse
+ DSYMBOL_DIR := dsymbol
+ 
+-SHELL:=/bin/bash
+ 
+ githash:
+-	@mkdir -p bin
+-	git describe --tags > bin/githash.txt
```

Comment:
The target doesn't have any dependencies, so why do we need to remove it?  Won't make consider it up to date because of the one in `postConfigure`, and then not bother running this?  Is it marked as PHONY somewhere?

Reply:
It gets run as it's a dependency of `ldcserver` and `ldcclient` which are dependencies of `ldc`. I'll just remove it as a dependency of those targets instead.